.github/workflows/tacos_detect_drift.yml

```
@@ -51,7 +51,8 @@ jobs:
       - id: slices
         run: |
           # NB: separate subprocess substitution, to avoid masking return values
-          slices="$("./tacos-gha/"lib/tacos/slices | jq -R | jq -sc)"
+          slices="$("./tacos-gha/"lib/tacos/slices \
+                      .config/tacos-gha/drift.allowlist | jq -R | jq -sc)"
```

nit:

Fixed-width indents only, please. As soon as this is no longer a one-liner, I'd continuate each `|`.

```suggestion
          slices="$(
            "./tacos-gha/"lib/tacos/slices \
                .config/tacos-gha/drift.allowlist |
              jq -R |
              jq -sc
            )"
```